src/utils/sortUtils.js

```diff
@@ -0,0 +1,6 @@
+export const groupArrayByKey = (array, key) => {
+    return array.reduce(function(rv, x) {
```

Comment:
Hvorfor `function(rv, x)` i stedet for `array.reduce((rv, x) => {})`?

Reply:
Sant, skal fikse det!

Reply:
Endet 👍